Devanshkanda.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Devansh kanda",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "i am a Tech Geek, a Software Developer with Security Mindset",
+  "avatar": "https://github.com/Devanshkanda.png",
```

Comment:
You can also remove this line, it is not required anymore :+1: 
```Suggestion
```

Reply:
Thanks @khushalarora for recommending changes, I have updated the file path and the changes you suggested.